hello-world/cli-and-ide-plugin/ide-plugin/build.gradle

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2020 The Arrow Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id "org.jetbrains.kotlin.jvm"
+    id "org.jetbrains.intellij" version "$INTELLIJ_GRADLE_PLUGIN_VERSION"
+}
+
+apply plugin: 'java'
+
+dependencies {
+    implementation ("io.arrow-kt:idea-plugin:$ARROW_META_VERSION") {
+        exclude group: "io.arrow-kt", module: "compiler-plugin"
+    }
+    implementation project(path: ':compiler-plugin', configuration: 'shadow')
+}
+
+buildSearchableOptions.enabled = false
+
+intellij {
+    version "$INTELLIJ_IDEA_VERSION"
+    pluginName = "Example"
+    plugins = ["gradle", "gradle-java", "java", "org.jetbrains.kotlin:${KOTLIN_IDEA_VERSION}", "io.arrow-kt.arrow:${ARROW_META_VERSION}"]
```

Comment:
Despite of having the IDE plugin "pending of approval" in Jetbrains repository, it can be resolved.

So `"io.arrow-kt.arrow:${ARROW_META_VERSION}"` refers to the plugin which was published 2 weeks ago.

**How to verify it?**

- Remove the compiler-plugin in `settings.gradle`
- Remove the dependencies in `ide-plugin/build.gradle`
- Remove OSS repository in `build.gradle`

The build check will continue working, because `"io.arrow-kt.arrow:${ARROW_META_VERSION}"` is being found in Jetbrains repository instead of OSS.

**Another check to verify it**, look at the content of the sandbox:

```
ide-plugin/build/idea-sandbox/plugins
├── Arrow Meta Intellij IDEA Plugin
│   └── lib
├── Example
│   └── lib
└── Kotlin
    ├── bin
    ├── kotlinc
    └── lib
```

The Arrow Meta plugin is called `Arrow Meta Intellij IDEA Plugin` and that was changed by `Arrow Meta` a few days ago.